data_visualization_in_seaborn/data_visualization_seaborn.md

```diff
@@ -0,0 +1,684 @@
+
+[( )] TRUE
+[(X)] FALSE
+****
+There are very few situations in which a scatter plot and line plot are equally appropriate.
+
+Line plots emphasize continuity and because of that are rarely used except for with data that have some kind of time component. Scatter plots are much more flexible and can work to show the relationship between almost any two continuous variables.  
+****
+
+What argument do you add to include facets in a `seaborn` plot?
+
+[[col, row]]
+<script>
+  let input = "@input".trim();
+  /col|row/i.test(input);
+</script>
+****
+You can add facets to a plot by including the arguments for `col`, `row`, or both.
+****
+
+## Trend Lines
+
+Trend lines look like line plots, but they are different in one key way: They show a **summary** of other data (usually a linear model), rather than plotting data directly.
+
+Trend lines are used to show the overall trend in a scatterplot. Sometimes, the scatterplot points themselves are omitted and just the trend lines are shown to keep the visualization as clean as possible, but they're still implied.
+
+### Linear regression trend lines
+
+Because trend lines are such a useful visual summary, `seaborn` provides several options to add trend lines to your plots quickly and easily.
+
+<div class = "options">
+There are two very similar functions available for drawing trend lines: `regplot` and `lmplot`. For many use cases, they are close to identical. For simplicity, we'll stick to `lmplot` in this lesson. Try subbing in `regplot` instead and make note of how the plots differ.
+
+For more background on `regplot` vs. `lmplot`, see [the seaborn regression tutorial](https://seaborn.pydata.org/tutorial/regression.html#controlling-the-size-and-shape-of-the-plot).
+</div>
+
+```python
+sns.lmplot(data = covid_data,
+            x="val_age", y="val_height_cm")
+```
+
+![Scatterplot with age on the x-axis and height on the y-axis, and a straight trend line running through the points, with a lighter shaded band around it.](media/seaborn_trend_1.png)
+
+A couple things about this plot are not ideal:
+
+1. Overplotting. The density of the scatter plot makes it hard to see the trend line as it runs through the bulk of the data.
+2. The linear trend doesn't appear to be a very good description of the relationship between age and height. It systematically over-estimates height at the very young ages, then under-estimates through early adulthood and over-estimates again late in life.
+
+We can address the first issue by adjusting alpha for the scatterplot. Alpha ranges between 0 (totally transparent) and 1 (totally opaque).
+
+<div class = "important">
+Tip: When your data overlap too much on a plot, use alpha to make them more transparent.
+</div>
+
+We face a problem here, though: Since we're using a single function to draw both the scatter plot and the trend line, how do we adjust alpha for the scatter plot without affecting the line?
```

Comment:
they haven't done anything in terms of adjusting alpha anywhere yet, have they? I'm not sure they would have a framework yet for why it would cause difficulty to adjust the alpha for just the plot. For all they know, perhaps there's an argument for line alpha vs scatterplot alpha in lmplot.

Reply:
I clarified this --- lemme know if you still think it's confusing.